ixofoundation/ixo-webclient
# feat(fuelentity): sign and submit

## Scope

https://nonaredteam.atlassian.net/browse/IXO-600

## Work Done

Refactoring of control panel
Reduxify fueling entity
Sign message for fueling an entity
Submit to api

## Steps to Test

Make sure you are logged in
Click on fuel project and enter random text
Bot will show summary screen
Click on submit and sign

NOTE: API is returning an error so the transaction won't go through
